src/components/Admin/CreateUser.jsx

```diff
@@ -114,54 +116,60 @@ function CreateUser({ exitModal }) {
         animate={{ opacity: 1, scale: 1 }}
         transition={{ duration: 0.6 }}
         exit={{ scale: 0.5 }}
-        >
+      >
         <button type="button" onClick={exitModal}>X</button>
-      <div className="admin-form">
-
-        <form onSubmit={handleSubmit}>
-        <h2>Create User</h2>
-          <label htmlFor="name">
-            Name: {" "}
-            <input
-              type="text"
-              id="name"
-              name="name"
-              value={name}
-              onChange={handleNameChange}
-              required
-            />
-          </label>
-
-          <label htmlFor="email">
-            Email: {" "}
-            <input
-              type="email"
-              id="email"
-              name="email"
-              value={email}
-              onChange={handleEmailChange}
-              required
-            />
-          </label>
-
-          <label htmlFor="roles">
-            Role: {" "}
-            <select
-              id="roles"
-              name="roles"
-              value={selectedRole}
-              onChange={handleRoleChange}
-              required
-            >
-              <option value="">Select a role</option>
-              {roles.map((role) => (
-                <option key={role.id} value={role.id}>{role.role}</option>
-              ))}
-            </select>
-          </label>
+        <div className="admin-form">
```

Comment:
Just food for thought for next time, these form elements could be their own react components